lgc/include/lgc/state/AbiUnlinked.h

```diff
@@ -95,6 +95,8 @@ static constexpr char FetchlessEsEntryName[] = "_amdgpu_es_main_fetchless";
 static constexpr char FetchlessHsEntryName[] = "_amdgpu_hs_main_fetchless";
 // Name of shader entry-point for LS that is fetchless VS
 static constexpr char FetchlessLsEntryName[] = "_amdgpu_ls_main_fetchless";
+// Name of shader entry-point for exportless PS
```

Comment:
Surely an exportless PS should have the same name as a normal PS. Code in ElfLinkerImpl::createGlueShaders uses the presence of the unlinked pal metadata to tell that it needs an export shader. The export shader should have the different name, because it is an epilogue rather than a prologue.